fluent/syntax/parser.py

```diff
@@ -30,15 +30,15 @@ def __init__(self, with_spans=True):
         self.with_spans = with_spans
 
     def parse(self, source):
-        ps = FTLParserStream(source)
-        ps.skip_blank_lines()
+        ps = FTLParserStream(source.replace('\r\n', '\n'))
```

Comment:
This will invalidate all spans including error positions for files with Windows line endings.

Looking at the js side of things, that doesn't handle Windows line endings at all right now?

I think we should use the algorithm we have in the reference parser to fix this issue, i.e., normalize newlines on text node creation.